package.json

```diff
@@ -107,6 +106,7 @@
   },
   "volta": {
     "node": "18.16.1",
-    "yarn": "1.22.21"
-  }
-}
\ No newline at end of file
+    "yarn": "1.22.22"
+  },
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```

Comment:
is this necessary?

Reply:
not strictly necessary, but it's a `package.lock` for your project's package manager.
it is defined (and followed) by corepack.

Reply:
corepack is being deprecated, i don't think we require it to be included as part of this PR change

Reply:
i wasn't aware of corepack deprecation (sad).
field removed.